src/commands/contract/view_storage/keys_to_view/keys_start_with_bytes_as_base64.rs

```diff
@@ -0,0 +1,33 @@
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = super::super::ViewStorageContext)]
+#[interactive_clap(output_context = KeysStartWithBytesAsBase64Context)]
+pub struct KeysStartWithBytesAsBase64 {
+    /// Enter the string that the keys begin with Base64 bytes (for example, "Uw=="):
+    keys_begin_with: String,
```

Comment:
Create a new type `Base64String` (you can take inspiration from `SignedTransactionAsBase64`), so base64 validation happens on input instead of in the context.

Reply:
Done